instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -25,7 +25,7 @@
 import org.slf4j.LoggerFactory;
 
 public abstract class BaseTracer {
-  private static final Logger log = LoggerFactory.getLogger(HttpServerTracer.class);
+  private static final Logger log = LoggerFactory.getLogger(BaseTracer.class);
```

Comment:
heh. this was also wrong and very confusing when I saw it in the logs while verifying. :)